### PR TITLE
docs(k3s): update quickstart documentation

### PR DESCRIPTION
**Pull request by vquiby**

### CODE REVIEW

File: docs/modules/ROOT/partials/k3s_kubeconfig_keycloak.adoc [hunk 1]
```diff
@@ -14,4 +14,8 @@ Retrieve the Keycloak password for the `admin` user of the `kubernetes` realm:
 $ terraform output admin_password
 ```
 
-You will use this user and password to log in to applications.
+You will use this user and password to log in:
+
+* Alertmanager
+* Keycloak
+* Prometheus
```
Comment (mcanevet) [inline docs/modules/ROOT/partials/k3s_kubeconfig_keycloak.adoc:21]: And also Grafana

File: modules/variables.tf [hunk 2]
```diff
@@ -65,13 +65,6 @@ variable "grafana_admin_password" {
   default     = null
 }
 
-variable "grafana_generic_oauth_extra_args" {
-  description = "override default oauth value(s)"
-  type        = map(string)
-  default = {
-    autoassign_org_role       = "Editor"
-  }
-}
```
Comment (mcanevet) [inline modules/variables.tf]: Why did you remove this?

**Review by mcanevet — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/partials/k3s_kubeconfig_keycloak.adoc (hunk 1 above)
Comment (vquiby) [reply docs/modules/ROOT/partials/k3s_kubeconfig_keycloak.adoc:21]: nope grafana admin user is not the same as the "admin user in Keycloak". 
I have added a section for Grafana & argoCD -> include::partial$k3s_argocd_grafana.adoc

File: modules/variables.tf (hunk 2 above)
Comment (vquiby) [reply modules/variables.tf]: It was my first try ... it does not work and it has nothing to do in a "doc" commit ;-)
My modification is in modules/values.tmpl.yaml

**Review by mcanevet — DISMISSED** [review]
